src/selectors/visibleEvents.js

```diff
@@ -0,0 +1,22 @@
+import moment from "moment";
+
+const visibleEvents = (events, { title, city, category, sortBy }) => {
+  return events
+    .filter(event => {
+      const isAvailable = moment().isSameOrBefore(event.endDate, "day");
+      const titleMatch = event.title.toLowerCase().includes(title.toLowerCase());
+      const cityMatch = city ? event.city === city : event;
+      const categoryMatch = category ? event.category === category : event;
+      return isAvailable && titleMatch && cityMatch && categoryMatch;
```

Comment:
`newline-before-return` could be also helpful 😅 
https://eslint.org/docs/rules/newline-before-return